scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
```

Comment:
You might also want to call rm -rf build_deps at the start of the install_dependencies script. If for some reason one of these functions in install_dependencies  errors out in the middle of a build, then the script will not be able to call mkdir build_deps as the folder will already exist.